usmqe_tests/api/ceph/test_ceph_cluster.py

```diff
@@ -112,11 +113,16 @@ def test_cluster_import_valid(valid_session_credentials):
         section="parameters")
 
     LOGGER.debug("integration_id: %s" % integration_id)
-    pytest.check(
-        [x for x in api.get_cluster_list()
-         if x["integration_id"] == integration_id],
-        "Job list integration_id '{}' should be "
-        "present in cluster list.".format(integration_id),
-        issue="https://github.com/Tendrl/api/issues/154")
+    try:
+        pytest.check(
+            [x for x in api.get_cluster_list() if x.get("integration_id", "") == integration_id],
```

Comment:
In that case, this test code would not catch a bug when multiple clusters with the same id are present in cluster list. In other words, I would rather store the list of clusters with given integration id into variable and then in `pytest.check()` call checked, that the len of the list is one. It's more explicit and covers the expected behavior better.